include/picongpu/simulation_defines/param/laser.param

```diff
@@ -369,6 +369,92 @@ namespace picongpu
         constexpr float_X LASER_PHASE = 0.0;
     }
 
+    namespace laserExpRampWithPrepulse
+    {
+    // Intensities of prepulse and ASE
+    constexpr float_X INT_RATIO_PREPULSE = 0.;
+    constexpr float_X INT_RATIO_POINT_1 = 1.e-8;
+    constexpr float_X INT_RATIO_POINT_2 = 1.e-4;
+    constexpr float_X INT_RATIO_POINT_3 = 1.e-4;
+
+    namespace SI
+    {
+        // time-positions of prepulse and ASE
+        constexpr float_64 TIME_PREPULSE_SI = 50.0e-15;
+        constexpr float_64 TIME_PEAKPULSE_SI = 1000.0e-15;
+        constexpr float_64 TIME_POINT_1_SI = 0.0;
+        constexpr float_64 TIME_POINT_2_SI = 700.0e-15;
+        constexpr float_64 TIME_POINT_3_SI = 900.0e-15;
+
+        /** unit: meter */
+        constexpr float_64 WAVE_LENGTH_SI = 0.8e-6;
+
+        /** UNITCONV */
+        constexpr float_64 UNITCONV_A0_to_Amplitude_SI = -2.0 * PI / WAVE_LENGTH_SI * ::picongpu::SI::ELECTRON_MASS_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI / ::picongpu::SI::ELECTRON_CHARGE_SI;
+
+        /** unit: W / m^2 */
+        // calculate: _A0 = 8.549297e-6 * sqrt( Intensity[W/m^2] ) * wavelength[m] (linearly polarized)
+
+        /** unit: none */
+        constexpr float_64 _A0  = 20.;
+
+        /** unit: Volt /meter */
+        constexpr float_64 AMPLITUDE_SI = _A0 * UNITCONV_A0_to_Amplitude_SI;
+
+        /** unit: Volt /meter */
+        //constexpr float_64 AMPLITUDE_SI = 1.738e13;
+
+        /** The profile of the test Lasers 0 and 2 can be stretched by a
+         *      constexprant area between the up and downramp
```

Comment:
typo:: constant

Reply:
looks like a failed auto replace of `const` to `constexpr` even though this should be `constant` - if there is such a wrong auto replacement here, there are probably more

Reply:
yes, obviously, but this was in the file before my edits
I changed the three occurrences of this